staging/src/k8s.io/client-go/tools/cache/mutation_detector.go

```diff
@@ -100,10 +99,7 @@ func (d *defaultCacheMutationDetector) AddObject(obj interface{}) {
 		return
 	}
 
-	copiedObj, err := scheme.Scheme.Copy(obj.(runtime.Object))
-	if err != nil {
-		return
-	}
+	copiedObj := obj.(runtime.Object).DeepCopyObject()
```

Comment:
Combine it with line 98 to avoid a type assertion.

Reply:
done